linalg/operator.hpp

```diff
@@ -462,20 +492,40 @@ class TimeDependentOperator : public Operator
 
    /** @brief Solve for the unknown @a k, at the current time t, the following
        equation:
-       F(@a u + @a gamma @a k, @a k, t) = G(@a u + @a gamma @a k, t).
+        1. $F( \widehat{u} , \dot{u}, t) = G( \widehat{u}, t)$, if solving for stage-state, $k = \widehat{u}$
+        2. $F( u + \gamma  \dot{u}, \dot{u}, t) = G( u +  \gamma \dot{u}, t)$, if solving for stage-slope, $ k = \dot{u} = \frac{du}{dt} $
+
+        where the stage state and slope follow the relation: $\dot{u} = \frac{du}{dt} = \frac{\widehat{u} - u}{\gamma}$.
 
        For solving an ordinary differential equation of the form
        $ M \frac{dy}{dt} = g(y,t) $, recall that F and G can be defined in
-       various ways, e.g.:
-
-         1. F(u,k,t) = k and G(u,t) = inv(M) g(u,t)
-         2. F(u,k,t) = M k and G(u,t) = g(u,t)
-         3. F(u,k,t) = M k - g(u,t) and G(u,t) = 0
-
-       Regardless of the choice of F and G, this function should solve for @a k
-       in M @a k = g(@a u + @a gamma @a k, t).
-
-       To see how @a k can be useful, consider the backward Euler method defined
+       various ways, depending on the stage variable:
+
+        <table>
+        <caption>Various definitions for $~F~$ and $~G~$</caption>
+        <tr>
+          <th>stage-state, $k = \widehat{u}$</th>
+          <th>stage-slope, $k = \dot{u} = \frac{du}{dt}$ </th>
+        </tr>
+        <tr>
+          <td>$F(\widehat{u},\dot{u},t) = \widehat{u}~$ and $~G(u,t) = \gamma M^{-1} g(u,t) + u$</td>
+          <td>$F(u,\dot{u},t) = \dot{u}~$ and $~G(u,t) = M^{-1} g(u,t)$</td>
+        </tr>
+        <tr>
+          <td>$F(\widehat{u},\dot{u},t) = M \widehat{u}~$ and $~G(u,t) = \gamma g(u,t) + M u$</td>
+          <td>$F(u,\dot{u},t) = M \dot{u}~$ and $~G(u,t) = g(u,t)$</td>
+        </tr>
+        <tr>
+          <td>$F(\widehat{u},\dot{u},t) = M \widehat{u} - \gamma g(\widehat{u},t)~$ and $~G(u,t) = M u$</td>
+          <td>$F(u,\dot{u},t) = M \dot{u} - g(u,t)~$ and $~G(u,t) = 0$</td>
+        </tr>
+        </table>
```

Comment:
Is it expected that the user has an HTML renderer active when viewing the code?  Asking to get an idea of the motivation for the HTML tags (aside from that I presume it looks nice in the rendered doxygen)

Reply:
Fixed/Removed in updated documentation as it's no longer needed